jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -478,143 +401,6 @@
               echo "Exiting with code: ${{rc}}"
               exit ${{rc}}
             fi
-
-      - build-test-federation-e2e-gce: # kubernetes-pull-build-test-federation-e2e-gce
-          status-context: Federation GCE e2e
-          max-total: 12
-          only-trigger-phrase: true  # This test should run only if explicitly triggered.
```

Comment:
I see that we are not setting this in the new file.
Based on the PR description, it seems like the intention was just moving the config, not changing it.
@spxtr  Is the change intentional?